scrapers/ep_votes/scrapers.py

```diff
@@ -570,7 +570,49 @@ def _format_paragraph(self, paragraph: Tag) -> str:
         return text
 
 
-class SessionsScraper(Scraper):
+class ParliamentSessionsScraper(Scraper):
+    BASE_URL = (
+        "https://www.europarl.europa.eu/plenary/en/ajax/"
+        "getSessionCalendar.html?family=PV&termId="
+    )
+
+    def __init__(self, term: int, year: int, month: int):
+        self.term = term
+        self.year = year
+        self.month = month
+
+    def _load(self) -> None:
+        self._resource = requests.get(self._url()).json()
+
+    def _url(self) -> str:
+        return f"{self.BASE_URL}{self.term}"
+
+    def _extract_data(self) -> List[Session]:
+        sessions = self._resource["sessionCalendar"]
+        # This assumes, that a session never spans over multiple months or years
+        sessionsInYear = [
```

Comment:
Code-Style Snake-Case

```suggestion
        sessions_in_year = [
```